scanner/config/config.go

```diff
@@ -80,7 +80,9 @@ type IndexerConfig struct {
 	// Enable if false disables the Indexer service.
 	Enable bool `yaml:"enable"`
 	// GetLayerTimeout timeout duration of GET requests for layers
-	GetLayerTimeout Duration `yaml:"get_layer_timeout"`
+	GetLayerTimeout Duration               `yaml:"get_layer_timeout"`
+	Repo            map[string]interface{} `yaml:"repo,omitempty" json:"repo,omitempty"`
```

Comment:
What is the reason for using `interface{}` here? I was expecting something like:

```golang
updater IndexerUpdater
```

Then:

```golang
type IndexerUpdater struct {
	Enabled bool `yaml:"enabled"`
	RepositoryToCPEURL string `yaml:"repository_to_cpe_url"`
}

It allows you to validate the URL, make changes, add local mappings (for running locally) or disable and pull from the internet, etc.

Reply:
yeah I agree. I don't think we need this level of indirection